Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs

```diff
@@ -5,35 +5,207 @@
 namespace Notatnik_parser.Notepad
 {
     public class NotepadMenu
-    {           
-        private void SaveFile(string textNoteParameter)
-        {            
-            string fileName = "LastNote.txt";
+    {    
+        string pathInstructions = "Podaj pełną ścieżkę z nazwą pliku lub samą nazwę pliku z rozszerzeniem, żeby {0} domyślnego folderu:";
+        
+        public void Run() {
+
+            ShowNotpadMenu();
+            MenageUserInput();
+        }
+
+        public void ShowNotpadMenu()
+        {
+            Console.WriteLine("Witamy w notatniku. Wybierz:");
+            Console.WriteLine("1 - nowa notatka");
+            Console.WriteLine("2 - otwórz notatkę");
+        }
+
+        private void MenageUserInput()
+        {
+            string insertValue;
+            do
+            {
+                insertValue = Console.ReadLine();
+
+            } while (!IsUserInputValid(insertValue));
+
+            GoToUserChoice(insertValue);
+        }
+
+        private bool IsUserInputValid(string userInput)
+        {
+            if (userInput == "1" || userInput == "2")
+            {
+                return true;
+            }
+
+            Console.WriteLine("Proszę wybrać numer z listy");
```

Comment:
move this somewere else. You have two dependencies: check input and show messages.